test/testcommon/src/test/java/io/pravega/test/common/JwtBodyTest.java

```diff
@@ -25,15 +24,13 @@ public void testSerialize() {
                 .issuedAtTime(Instant.now().getEpochSecond())
                 .expirationTime(Instant.now().plusSeconds(50).getEpochSecond())
                 .build();
-
-        String json = new Gson().toJson(jwtBody);
-        assertNotNull(json);
+        assertNotNull(jwtBody.toString());
```

Comment:
can this 'toString()' return a null value ?

Reply:
I've modified the assertions to make them more relevant, given the change.